lib/cleanup.js

```diff
@@ -11,16 +11,35 @@ module.exports = {
     const cli = this.options.verbose ? this.serverless.cli : { log: _.noop };
 
     if (!keepOutputDirectory) {
-      cli.log(`Remove ${webpackOutputPath}`);
+      if (this.log) {
+        this.log.verbose(`Remove ${webpackOutputPath}`);
+      } else {
+        cli.log(`Remove ${webpackOutputPath}`);
+      }
       if (this.serverless.utils.dirExistsSync(webpackOutputPath)) {
         // Remove async to speed up process
         fse
           .remove(webpackOutputPath)
-          .then(() => cli.log(`Removing ${webpackOutputPath} done`))
-          .catch(error => cli.log(`Error occurred while removing ${webpackOutputPath}: ${error}`));
+          .then(() => {
+            if (this.log) {
+              this.log.verbose(`Removing ${webpackOutputPath} done`);
+            } else {
+              cli.log(`Removing ${webpackOutputPath} done`);
+            }
+            return null;
```

Comment:
Do we really need to return something?

Reply:
Unfortunately ESLint is setup in that way:

![Screenshot 2021-12-01 at 15 45 29](https://user-images.githubusercontent.com/122434/144255432-b601e9df-a331-4dee-b982-4be888201a12.png)